### DISCUSSION TITLE
Problem with SPISW in arduino Platforme

### DISCUSSION THREAD

**Discussion by droma28** [Seeed-Solution/SenseCAP_Indicator_ESP32#59] [Q&A]
hello everyone, I have a problem that I don't understand, when I program via ESP-IDF I don't have a problem, but when I want to use Arduino IDE, my program works well but when I turn off the screen , my program runs well (serial trace) but the screen remains black, so that I return to normal operation I re-flash again via ESP-IDF,, I think this problem came to SPISW with CS in I2C , Do you have any ideas  

Sincerely

**Comment by Love4yzp** [comment]
https://github.com/Seeed-Studio/wiki-documents/issues/1403

hope to have your contribution.
